website/components/pluginData.tsx

```diff
@@ -1,226 +1,283 @@
     href: "https://github.com/simpleanalytics/cq-source-simpleanalytics",
-    id: "simpleanlaytics",
+    id: "simpleanalytics",
   },
   {
-    title: "Slack",
     name: "Slack",
     logo: "/images/logos/plugins/slack.svg",
     id: "slack",
   },
   {
-    title: "Snyk",
     name: "Snyk",
     logo: "/images/logos/plugins/snyk.svg",
     id: "snyk",
   },
   {
-    title: "Stripe",
     name: "Stripe",
     logo: "/images/logos/plugins/stripe.svg",
     id: "stripe",
   },
   {
-    title: "Tailscale",
     name: "Tailscale",
     logo: "/images/logos/plugins/tailscale.svg",
     logoDark: "/images/logos/plugins/tailscale-dark.svg",
     id: "tailscale",
   },
   {
-    title: "Terraform",
     name: "Terraform",
     logo: "/images/logos/plugins/terraform.svg",
     id: "terraform",
   },
   {
-    title: "Vercel",
     name: "Vercel",
     logo: "/images/logos/plugins/vercel.svg",
     logoDark: "/images/logos/plugins/vercel-dark.svg",
     id: "vercel",
   },
   {
-    title: "Yandex Cloud",
     name: "Yandex Cloud",
     logo: "/images/logos/plugins/yandex.svg",
     id: "yandexcloud",
     href: "https://github.com/yandex-cloud/cq-source-yandex",
   },
   {
-    title: "And more...",
     name: "All Plugins",
     logo: "/images/logos/plugins/more.svg",
     logoDark: "/images/logos/plugins/more-dark.svg",
+    id: "more",
     href: "/docs/plugins/overview",
   },
 ];
+
+export const DESTINATION_PLUGINS: Plugin[] = [
+  {
+    name: "Azure Blob Storage",
+    logo: "/images/logos/plugins/azblob.svg",
+    id: "azblob",
+  },
+  {
+    name: "BigQuery",
+    logo: "/images/logos/plugins/bigquery.svg",
+    id: "bigquery",
+  },
+  {
+    name: "ClickHouse",
+    logo: "/images/logos/plugins/clickhouse.svg",
+    id: "clickhouse",
+  },
+  {
+    name: "Elasticsearch",
+    logo: "/images/logos/plugins/elasticsearch.svg",
+    id: "elasticsearch",
+  },
+  {
+    name: "File",
+    logo: "/images/logos/plugins/file.svg",
+    id: "file",
+  },
+  {
+    name: "GCS",
+    logo: "/images/logos/plugins/gcs.svg",
+    id: "gcs",
+  },
+  {
+    name: "Kafka",
+    logo: "/images/logos/plugins/kafka.svg",
+    id: "kafka",
+  },
+  {
+    name: "Microsoft SQL Server",
+    logo: "/images/logos/plugins/mssql.svg",
+    id: "mssql",
+  },
+  {
+    name: "MongoDB",
+    logo: "/images/logos/plugins/mongodb.svg",
+    id: "mongodb",
+  },
+  {
+    name: "Neo4j",
+    logo: "/images/logos/plugins/neo4j.svg",
+    id: "neo4j",
+  },
+  {
+    name: "PostgreSQL",
+    logo: "/images/logos/plugins/postgresql.svg",
+    id: "postgresql",
+  },
+  {
+    name: "S3",
+    logo: "/images/logos/plugins/s3.svg",
+    id: "s3",
+  },
+  {
+    name: "Snowflake",
+    logo: "/images/logos/plugins/snowflake.svg",
+    id: "snowflake",
+  },
+  {
+    name: "SQLite",
+    logo: "/images/logos/plugins/sqlite.svg",
+    id: "sqlite",
+  },
+];
+
+export const ALL_PLUGINS: Plugin[] = Array.prototype.concat(SOURCE_PLUGINS, DESTINATION_PLUGINS);
```

Comment:
```suggestion
export const ALL_PLUGINS: Plugin[] = [...SOURCE_PLUGINS, ...DESTINATION_PLUGINS];
```

Is a new shorter syntax for array concatenation